src/main/scala/nl/knaw/dans/easy/stage/fileitem/EasyFileMetadata.scala

```diff
@@ -15,13 +15,18 @@
  */
 package nl.knaw.dans.easy.stage.fileitem
 
+import java.io.File
+
 import scala.util.Try
 
 object EasyFileMetadata {
   def apply(s: FileItemSettings): Try[String] = Try {
+      val parentPath = s.pathInDataset.get.getParentFile
```

Comment:
Do you mean 

     val parentPathOption = s.pathInDataset.map(_.getParentFile)

I am not sure I see the advantage in that. It will still need to throw in the event that `s.pathInDataset` is `None`. The `Try` constructor will catch the exception, a useful technique that @GeorgiKhomeriki always used ;) Maybe I am missing something ...